aws_auth.tf

```diff
@@ -53,7 +53,7 @@ resource "kubernetes_config_map" "aws_auth" {
   data = {
     mapRoles    = <<EOF
 ${join("", distinct(concat(data.template_file.launch_template_worker_role_arns.*.rendered, data.template_file.worker_role_arns.*.rendered)))}
-${yamlencode(var.map_roles)}
+%{if var.map_roles == []}${yamlencode(var.map_roles)}%{endif}
```

Comment:
Are you sure this is correct? It looks like it's only doing the `yamlencode` if the var is []. Maybe should be `!=`?

Reply:
@dpiddockcmp Sorry, silly copy paste error